lang_java/parsing/tree-sitter-parser

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+example_count=$(find $1 -name '*.java' | wc -l)
```

Comment:
the code looks almost identical to json-generate.
Maybe you could factorize code and just have flag to differentiate.